githubactions-db/files/etc/mysql/conf.d/custom.cnf

```diff
@@ -1,26 +1,31 @@
 
 [mysqld]
+# Tweak optimizer
+optimizer_switch='mrr=on'
+optimizer_switch='mrr_cost_based=off'
+
+# Mount datadir onto memory
+datadir=/dev/shm/mysql
 
-# Performance tweaks
-# see https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/mysql/images/*/Dockerfile
-# see https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/mysql/images/*/ram/Dockerfile
-collation-server               = utf8mb4_unicode_ci
-init-connect                   = "SET NAMES utf8mb4"
-character-set-server           = utf8mb4
-innodb_flush_log_at_trx_commit = 2
-sync_binlog                    = 0
-innodb_use_native_aio          = 0
```

Comment:
This does not really affect performances. Indeed, as datadir is mounted onto memory, we are not really affected by configuration sthat acts on disk writing.